runtime-network/neatvnc/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=neatvnc
+PKGSEC=net
+PKGDES="A liberally licensed VNC server library with a clean interface"
```

Comment:
This means very little to the end user - try this:

```
A fast VNC server library with a clean API interface
```

Reply:
Fixed.